### PR TITLE
Add configuration to logging in WebHost

### PR DESCRIPTION
**Pull request by BrennanConroy**
#45

### CODE REVIEW

File: src/Microsoft.AspNetCore/WebHost.cs [hunk 1]
```diff
@@ -173,8 +173,9 @@ public static IWebHostBuilder CreateDefaultBuilder(string[] args)
                         config.AddCommandLine(args);
                     }
                 })
-                .ConfigureLogging(logging =>
+                .ConfigureLogging((hostingContext, logging) =>
                 {
+                    logging.UseConfiguration(hostingContext.Configuration.GetSection("Logging"));
```
Comment (Eilon) [inline src/Microsoft.AspNetCore/WebHost.cs]: Can we add a test for this?

Comment (BrennanConroy) [inline src/Microsoft.AspNetCore/WebHost.cs]: There aren't any tests in MetaPackages yet :'(
See https://github.com/aspnet/MetaPackages/pull/46

**Comment by DamianEdwards** [comment]
Looks good

File: src/Microsoft.AspNetCore/WebHost.cs (hunk 1 above)
Comment (JunTaoLuo) [inline src/Microsoft.AspNetCore/WebHost.cs]: #46 has been merged, this should be unblocked.

Comment (Eilon) [inline src/Microsoft.AspNetCore/WebHost.cs]: @BrennanConroy can we get a basic test for this added so we can get this into Preview1?

Comment (BrennanConroy) [inline src/Microsoft.AspNetCore/WebHost.cs]: I'm adding one!

File: test/Microsoft.AspNetCore.FunctionalTests/WebHostFunctionalTests.cs [hunk 2]
```diff
@@ -200,6 +200,43 @@ public WebHostFunctionalTests(ITestOutputHelper output) : base(output)
...
+
+                    logger.Log(LogLevel.Information, 0, "Message", null, (s, e) =>
+                    {
+                        Assert.True(false);
+                        return string.Empty;
+                    });
+
+                    logger.Log(LogLevel.Warning, 0, "Message", null, (s, e) =>
+                    {
+                        return string.Empty;
```
Comment (Eilon) [inline test/Microsoft.AspNetCore.FunctionalTests/WebHostFunctionalTests.cs]: How can we tell that this log *did* get written out? Can either use mocks, or at least set a flag to `true` or something and then assert that.

File: test/TestSites/CreateDefaultBuilderApp/Program.cs [hunk 3]
```diff
@@ -77,34 +76,11 @@ private static string GetResponseMessage(WebHostBuilderContext context, IService
...
-            }
-
-            // Verify AddDebug called
-            if (!testLoggerFactory.Providers.Any(provider => provider is DebugLoggerProvider))
-            {
-                return $"Debug logger not added to ILoggerFactory.";
-            }
-
+            // TODO: Verify AddConsole called
+            // TODO: Verify AddDebug called
```
Comment (BrennanConroy) [inline test/TestSites/CreateDefaultBuilderApp/Program.cs]: https://github.com/aspnet/Logging/issues/609

**Comment by BrennanConroy** [comment]
:up: 📅

**Pull request merged**